gradle.properties

```diff
@@ -15,7 +15,7 @@
 #
 
 ideaEdition = IU
-ideaVersion = 2018.2
+ideaVersion = 2018.3
```

Comment:
@ivanporty was this intentional? I guess it makes sense now that its GA.

Also, I think this is why the build is failing - I need to add the artifacts to our GCS bucket cache.